### ISSUE TITLE
dexie-relationships do not work with require in electron

### ISSUE THREAD

**Issue by codeur4** [ignasbernotas/dexie-relationships#6]
dexie-relationships do not work with require in electron, but dexie alone does
```
var Dexie = require('dexie')
var relationships = require('dexie-relationships')
var db = new Dexie('MyProject', {addons: [relationships]})
```
Line 3 is in error.
Here is the error comming in the node_modules/dexie/dist/dexie.js file :

![dexieerror](https://cloud.githubusercontent.com/assets/10487632/20580197/4784ea88-b1d1-11e6-86e1-e6c0a5b4c6b1.jpg)

dexie.js works well alone, so I suggest it may come from the webpack formatting ?

![dexieerror2](https://cloud.githubusercontent.com/assets/10487632/20580317/f5f910d0-b1d1-11e6-995f-0d6d0455e02b.jpg)

Thanks for help

**Comment by ignasbernotas** [comment]
Hi, yeah, the distributable files need fixing. I'll take a look at this 1st thing on Monday.

**Comment by codeur4** [comment]
Ok ty, let me know when you find ou something

**Comment by ignasbernotas** [comment]
Hi, this has been fixed with 1.2.0 :) Closing the issue, but let me know how you get on. Thanks!
